flows/contact.go

```diff
@@ -11,17 +11,32 @@ import (
 
 // Contact represents a single contact
 type Contact struct {
-	uuid     ContactUUID
-	name     string
-	language utils.Language
-	timezone *time.Location
-	urns     URNList
-	groups   GroupList
-	fields   *Fields
+	uuid           ContactUUID
+	name           string
+	language       utils.Language
+	otherLanguages utils.LanguageList
```

Comment:
I started off with one list this morning (https://github.com/nyaruka/rapidpro/pull/1347) and then switch it to two because one list doesn't let a user determine whether a contact has a language stored, i.e. we need to differentiate between the explicitly set language and implicit backdown languages.